pyam/core.py

```diff
@@ -392,20 +393,60 @@ def validate(self, criteria={}, exclude=False):
         """
         df = _apply_criteria(self.data, criteria, in_range=False)
 
-        if exclude:
-            idx = _meta_idx(df)
-            self.meta.loc[idx, 'exclude'] = True
-
         if not df.empty:
             msg = '{} of {} data points to not satisfy the criteria'
             logger().info(msg.format(len(df), len(self.data)))
 
-            if exclude and len(idx) > 0:
-                logger().info('{} non-valid scenario{} will be excluded'
-                              .format(len(idx), '' if len(idx) == 1 else 's'))
+            if exclude:
+                self._exclude(df)
 
             return df
 
+    def check_aggregate(self, var_total, var_sector=None, units=None,
+                        exclude=False, threshold=0.1, multiplier=1):
```

Comment:
Is there a use case for the multiplier?

Reply:
one use case in the SR1.5 context is to check that values are equal but with different signs (like CCS quantity vs. carbon removed must be equal, but one is usually reported as positive and the other as negative). Or going forward, making sure that timeseries in different units are equal...